_fossfund/13-optikey.md

```diff
@@ -0,0 +1,10 @@
+---
+title: OPTIKEY
+logo: optikey.png
```

Comment:
[Place this image  ](https://github.com/OptiKey/OptiKey/raw/gh-pages/images/OptiKey-logo-600x200.png)

in  https://github.com/microsoft/opensource.microsoft.com/tree/1dbc9603ccbda0f41c0535267ca502abe80aa0aa/assets/images/fossfund